saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/content/ImportContentFileUtils.java

```diff
@@ -0,0 +1,84 @@
+package pl.edu.icm.saos.importer.notapi.common.content;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.commons.io.FileUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import pl.edu.icm.saos.importer.common.ImportException;
+import pl.edu.icm.saos.persistence.model.importer.RawSourceJudgment;
+
+/**
+ * Utilities for operations on content files used in download process.
+ * 
+ * @author madryk
+ */
+@Service
+public class ImportContentFileUtils {
```

Comment:
Probably this class can be removed if you get rid of the need of using rawSourceJudgmentClass.